app/_src/gateway/kong-plugins/queue/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: About Plugin Queuing 
+content_type: explanation
+---
+
+Starting with {{site.base_gateway}} 3.3.x, some Kong plugins now use queuing to decouple the production of data in the proxy path and the submission of that data to a server, such as a log server. This ensures that the upstream server isn't taxed by recieving many proxy requests at the same time.
+
+The following plugins now use queuing:
+
+* [HTTP Log](/hub/kong-inc/http-log/)
+* [OpenTelemetry](/hub/kong-inc/opentelemetry/)
+* [Datadog](/hub/kong-inc/datadog/)
+* [StatsD](/hub/kong-inc/statsd/)
+* [Zipkin](/hub/kong-inc/zipkin/)
+
+This page describes why queues are needed and how plugin queuing works.
+
+## Why are queues needed? 
+
+Some analytic and monitoring plugins, like HTTP Log and Datadog, must send request information that was processed by {{site.base_gateway}} to another server (the upstream server). 
+
+Without queuing, the information is sent during the log phase to avoid introducing any latency to request or response. This isn't ideal because directly sending data from the log handler to the upstream server creates a large number of requests when {{site.base_gateway}} is under a high load. Sending out a large number of concurrent outbound requests can overload the log server and can negatively impact the ability of the machine running {{site.base_gateway}} to proxy requests.
```

Comment:
```suggestion
Sending information directly from the log handler to the upstream server, instead of queuing it, avoids introducing latency to requests and responses. However, this approach can create a large number of requests when {{site.base_gateway}} is under a high load, which can overload the log server and negatively impact the ability of the machine running {{site.base_gateway}} to proxy requests.
```